crates/pessimistic-proof-test-suite/src/forest.rs

```diff
@@ -193,6 +201,7 @@ impl Forest {
         let new_local_exit_root = self.local_exit_tree.get_root();
         let (imported_exits_root, signer, signature) =
             compute_signature_info(new_local_exit_root, &imported_bridge_exits);
+        let dummy = self.local_exit_tree_data_a.get_root();
```

Comment:
Would `imported_rollup_exit_root` be a better name here?

Reply:
I wanted to emphasize that it isn't used here in the test suite since it deals only with the mainnet exit tree for now.

I removed it to put directly a default instead.